python/pyarrow/tests/test_ipc.py

```diff
@@ -51,16 +51,16 @@ def get_source(self):
 
     def write_batches(self, num_batches=5, as_table=False):
         nrows = 5
-        schema = pa.schema([('one', pa.float64()), ('two', pa.utf8())])
+        schema = pa.schema([("one", pa.float64()), ("two", pa.utf8())])
```

Comment:
Any reason for all these style changes? These don't seem related. Did you apply a formatting tool by mistake?